RoddyCore/src/de/dkfz/roddy/execution/jobs/Job.groovy

```diff
@@ -333,6 +338,100 @@ class Job extends BEJob<BEJob, JobResult> {
         path
     }
 
+
+    /**
+     * Stores a new job jobState info to an execution contexts job jobState log file.
+     *
+     * @param job
+     */
+    void appendToJobStateLogfile(BatchEuphoriaJobManager jobManager, ExecutionContext executionContext, BEJobResult res, OutputStream out = null) {
+        // Ugly, but works. Java/Groovy's missing runtime dispatch on method arguments sucks.
+        if (jobManager instanceof ClusterJobManager) {
+            appendToJobStateLogfileForCluster(executionContext, res)
```

Comment:
I don't like the idea here, that we need instanceof. This should be handled by Roddys ExecutionService (Local/SSH). There is the writeLine method or so. Or a pattern like that for Strategy or Visitor depending on the Roddy execution service might be good?